assets/css/main.css

```diff
@@ -58,6 +58,27 @@ table.src tr td.src {
   user-select: none;
 }
 
+/* focus styles (component-specific focus styles can be included in other 
+   components) */
+
+a:focus { /* Mostly for links in text */
+    background-color: #ffdd00 !important;
+    color: #333 !important;
+    outline: 3px solid rgba(0, 0, 0, 0);
+    text-decoration: underline;
+}
+
+a.btn:focus, button:focus, input.btn:focus {
+    background-color: #ffdd00 !important;
+    color: #000 !important;
+    border: 2px solid #33 !important;
```

Comment:
Was this border color changed to `#33` (from `#333`) on purpose?

Reply:
It was not! Thanks!